Mypy error in LiteLLM: Ellipses cannot accompany other argument types in function type signature

### Description

Mypy complains with the following error:
```
.pyenv/versions/3.11.7/envs/ai-conversations-api-env/lib/python3.11/site-packages/mirascope/core/litellm/_utils/_setup_call.py:26: error: Ellipses cannot accompany other argument types in function type signature  [syntax]
.pyenv/versions/3.11.7/envs/ai-conversations-api-env/lib/python3.11/site-packages/mirascope/core/litellm/_utils/_setup_call.py:48: error: Ellipses cannot accompany other argument types in function type signature  [syntax]
```

The problem is the `client: ...` part in this code block:
```python
@overload
def setup_call(
    *,
    model: str,
    client: ...,
    fn: Callable[..., Awaitable[AsyncOpenAIDynamicConfig]],
    fn_args: dict[str, Any],
    dynamic_config: AsyncOpenAIDynamicConfig,
    tools: list[type[BaseTool] | Callable] | None,
    json_mode: bool,
    call_params: OpenAICallParams,
    extract: bool,
) -> tuple[
    AsyncCreateFn[ChatCompletion, ChatCompletion],
    str | None,
    list[ChatCompletionMessageParam],
    list[type[OpenAITool]] | None,
    OpenAICallKwargs,
]: ...
```

The change was introduced in [this commit](https://github.com/Mirascope/mirascope/commit/31a229e5b0deb6905fe6438aa3b04edd100e9174), released in [version 1.7.0](https://github.com/Mirascope/mirascope/releases/tag/v1.7.0).

I couldn't make Mypy ignore the error, so I had to downgrade to Mirascope 1.6.1 in order to make it work.


### Python, Mirascope & OS Versions, related packages (not required)

```TOML
Python 3.11.7
Mirascope 1.18.3
Mypy 1.15.0
```

--- a/mirascope/core/litellm/_utils/_setup_call.py
+++ b/mirascope/core/litellm/_utils/_setup_call.py
@@ -1,7 +1,7 @@
 """This module contains the setup_call function for OpenAI tools."""
 
 from collections.abc import Awaitable, Callable
-from typing import Any, cast, overload
+from typing import Any, TypeAlias, cast, overload
 
 from litellm import acompletion, completion
 from openai import OpenAI
@@ -21,12 +21,16 @@ from ...openai import (
 from ...openai._call_kwargs import OpenAICallKwargs
 from ...openai._utils import setup_call as setup_call_openai
 
+# Note: MyPy doesn't like `client: ...` so we use these aliases instead.
+_AsyncLiteLLMClient: TypeAlias = Any
+_SyncLiteLLMClient: TypeAlias = Any
+
 
 @overload
 def setup_call(
     *,
     model: str,
-    client: ...,
+    client: _AsyncLiteLLMClient | None,
     fn: Callable[..., Awaitable[AsyncOpenAIDynamicConfig]],
     fn_args: dict[str, Any],
     dynamic_config: AsyncOpenAIDynamicConfig,
@@ -48,7 +52,7 @@ def setup_call(
 def setup_call(
     *,
     model: str,
-    client: ...,
+    client: _SyncLiteLLMClient | None,
     fn: Callable[..., OpenAIDynamicConfig],
     fn_args: dict[str, Any],
     dynamic_config: OpenAIDynamicConfig,

